spring-cloud-azure-samples/spring-cloud-azure-storage-queue-integration-sample/src/main/java/example/WebController.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package example;
+
+import com.microsoft.azure.spring.integration.core.AzureMessageHandler;
+import com.microsoft.azure.spring.integration.core.api.CheckpointMode;
+import com.microsoft.azure.spring.integration.storage.queue.StorageQueueOperation;
+import com.microsoft.azure.spring.integration.storage.queue.inbound.StorageQueueMessageSource;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.integration.annotation.InboundChannelAdapter;
+import org.springframework.integration.annotation.MessagingGateway;
+import org.springframework.integration.annotation.Poller;
+import org.springframework.integration.annotation.ServiceActivator;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageChannel;
+import org.springframework.messaging.MessageHandler;
+import org.springframework.util.concurrent.ListenableFutureCallback;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ * @author Miao Cao
+ */
+@RestController
+    public class WebController {
+    /*Storage queue name can only be made up of lowercase letters, the numbers and the hyphen(-).*/
+    private static final String STORAGE_QUEUE_NAME = "example";
+    private static final String OUTPUT_CHANNEL = "outputChannel";
+    private static final String INPUT_CHANNEL = "inputChannel";
+    private static final Log LOGGER = LogFactory.getLog(StorageQueueIntegrationApplication.class);
```

Comment:
Incorrect class name for the LOGGER? Move LOGGER to the first line?

Reply:
Use Slf4j for the LOGGER? To align with others.

Reply:
Some used logging and some used slf4j. I will change the class name for LOGGER first.